### DISCUSSION TITLE
No daytime image

### DISCUSSION THREAD

**Discussion by Mooflotic** [aaronwmorris/indi-allsky#1365] [General]
Hi Aaron, 
I have daytime capture ON but daytime timelapse OFF and there's no image in the Latest tab, it just says "Daytime timelapse disabled. Night starts in 12.2 hours." is this correct? I was expecting an updated (but then discarded) image.

I've seen a different thread here but it was for a remote website. Not my case.

**Comment by aaronwmorris** [comment]
Is it possible you changed this setting and did not reload yet?  It should be showing an image.

Can you post some messages from `/var/log/indi-allsky/indi-allsky.log` ?

**Reply by Mooflotic** [reply]
Fixed, it was due to the disk beeing full...my fault.
